Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT"
 homepage = "https://github.com/sdroege/async-tungstenite"
 repository = "https://github.com/sdroege/async-tungstenite"
 documentation = "https://docs.rs/async-tungstenite"
-version = "0.29.0"
+version = "0.29.1"
```

Comment:
`Cargo.lock.msrv` needs to be updated for this too

Reply:
Ok, I'll do that then. Not a problem (copy the file, `cargo +1.64 check`, copy the updated file back)

Reply:
@sdroege I look forward to that becoming less necessary once the MSRV is sufficiently new to allow using the MSRV-aware resolver. :)